tests/test_computed_fields.py

```diff
@@ -815,3 +815,26 @@ def my_field_serializer(self, value: Any, info: FieldSerializationInfo) -> Any:
             return f'{info.field_name} = {value}'
 
     assert MyModel().model_dump() == {'my_field': 'my_field = foo', 'other_field': 'other_field = 42'}
+
+
+def test__fields_on_instance_and_cls() -> None:
```

Comment:
```suggestion
def test_fields_on_instance_and_cls() -> None:
```

or maybe it is intentional

Reply:
Good catch, thanks